server/utils/mpesa_utils.py

```
@@ -0,0 +1,278 @@
+            if result_code == 0:
+                transaction.status = 'completed'
+                if transaction_code:
+                    transaction.transaction_code = transaction_code
+            else:
+                transaction.status = 'failed'
+
+            db.session.commit()
+
+            return transaction
+
+        except Exception as e:
```

Flake8 (7.2.0)

[error] local variable 'e' is assigned to but never used

(F841)